tests/system/test_query.py

```diff
@@ -377,6 +377,7 @@ def test_query_default_namespace_when_context_namespace_is_other(
     class SomeKind(ndb.Model):
         foo = ndb.IntegerProperty()
         bar = ndb.StringProperty()
+        discriminator = ndb.StringProperty(default=namespace)
```

Comment:
I'm a little confused as to why we're using `namespace` here to populate a field value:  is it just because we know it to be unique per run?  ISTM it would be clearer to create another value, using `str(uuid.uuid4())` (or make a `discriminator` fixture which does that, if you are contemplating using the value in other testcases).

Reply:
Yes, that is the reason. I did waffle a bit on creating a new value for clarity versus using an already available value. You just made the clarity argument, so I'll switch to that. 9228131